docs/API.md

```diff
@@ -63,6 +63,26 @@ This options can be applied in addition to the command options when numbers are
 | <code><var>lte</var></code> | number | less then equals |
 | <code><var>gte</var></code> | number | greater than or equals |
 
+### Handling HTML Entities
+
+An HTML entity is a piece of text (“string”) that begins with an ampersand (&) and ends with a semicolon (;). 
+Entities are frequently used to display reserved characters (which would otherwise be interpreted as HTML code), 
+and invisible characters (like non-breaking spaces &nbsp;).
```

Comment:
```suggestion
An HTML entity is a piece of text (“string”) that begins with an ampersand (`&`) and ends with a semicolon (`;`). Entities are frequently used to display reserved characters (which would otherwise be interpreted as HTML code), and invisible characters (like non-breaking spaces, e.g. `&nbsp;`).
```